library/Process-linux.cpp

```diff
@@ -220,7 +219,8 @@ bool Process::setPermissions(const t_memrange & range,const t_memrange &trgrange
 
 bool Process::flushCache(const void* target, size_t count)
 {
-    return cacheflush(target, count, BCACHE);
+    __builtin___clear_cache((char*)target, (char*)target + count - 1);
+    return false; /* assume always succeeds, as the builtin has no return type */
```

Comment:
Just verifying, you return false for success?

Reply:
yeah, following the unix standard of 0 (false) for success and non-zero for failure. this is used elsewhere in DFHack for OS API calls (see, e.g. `PlugLoad`)

Reply:
let me be clear that i don't have a problem with returning true for success :)

Reply:
It seems to me that returning 0 for success makes sense when the return value is an int, but if it's a bool, then it should return true for success. If this matches the semantics of the surrounding functions and is consistent across the different implementations in Process-linux and Process-windows, though, I'm fine with it.

Reply:
switched, since tbh i prefer returning `true` on success as well